recipes/holidays/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "holidays" %}
+{% set version = "0.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b38e11736e2fad04c51e6d8d38ff66b483905d9448ba0291640cda5889687baa
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - python-dateutil
+    - six
```

Comment:
Only dependencies need to install the package should appear here. This is anything needed by the setup.py.